qcodes/instrument_drivers/QuTech/IVVI.py

```diff
@@ -260,17 +338,15 @@ def read(self, message_len=None):
 
         while bytes_in_buffer < message_len:
             t1 = time.time()
-            time.sleep(.05)
+            time.sleep(.025)
```

Comment:
Not a fan of hardcoded sleep times but I believe this is necessary for the IVVI.